local/languages/finna/en-gb.ini

```diff
@@ -771,37 +769,59 @@ Publications = "Publications"
 Publisher or Distributor Number = "Publisher or Distributor Number"
 Publish date = "Publish date"
 Published In Issue = "Published in"
-R2_accessrights_info_html = ""
-R2_accessrights_menu_heading = ""
-R2_accessrights_register = ""
-R2_accessrights_blocklisted_html = ""
-R2_accessrights_session_expired = ""
-R2_menu_heading_register = ""
-R2_register_error = ""
-R2_register_form_datasource = ""
-R2_register_form_title = ""
-R2_register_form_help_post_html = ""
-R2_register_form_help_pre_html = ""
-R2_register_form_help_returninguser_pre_html = ""
-R2_register_form_userinfo = ""
-R2_register_form_age = ""
-R2_register_form_license_accept = ""
-R2_register_form_usage = ""
-R2_register_form_usage_help_html = ""
-R2_register_form_usage_help_tooltip_html = ""
-R2_register_form_usage_option1 = ""
-R2_register_form_usage_option2 = ""
-R2_register_form_usage_option3 = ""
-R2_register_form_usage_option4 = ""
-R2_register_form_usage_option5 = ""
-R2_register_form_usage_option6 = ""
-R2_register_myresearch = ""
-R2_rems_connect_error = ""
-R2_rems_session_expired = ""
-R2_search_title = ""
-R2_session_usage_purpose = ""
-R2_termsofuse_content_html = ""
-R2_your_accessrights = ""
+R2_accessrights_title = "Access rights to Finna Reading Room"
```

Comment:
Onko "Reading Room" nyt Tutkijasalin virallinen käännös? Voin nähdä tämän sotkeutuvan "Lukusaliin", joka voisi olla vähän erilainen palvelu..
"Research Room"?

Reply:
Tutkijasali (arkistojen lukusali) on englanniksi Reading Room

Reply:
En muuta väitä, mutta koska tuo on brandatty "Finna Reading Room", niin ei sitten voida toteuttaa "Finna Lukusali" -palvelua myöhemmin muulle aineistolle.

Reply:
Monet arkistot näkyvät käyttävän termiä "Archives Reading Room", joka sitten on yksiselitteinen myös arkiston ulkopuolella. Näen tässä siis ongelmana sen, että Finna ei ole vain arkistoja varten, ja tämä nimeäminen sitoo lukusali-käsitteen arkistoihin.

Reply:
mun mielestä ei sido arkistoihin, koska samaa voi käyttää kirjastoillekin

Reply:
Jos se olisi "Finna Lukusali" suomeksi ja vastaava ruotsiksi, mutta tutkijasali ei sitten taas toimikaan samalla tavalla varsinkaan yleiselle kirjastolle.